Nure/Configuration/NureOptions.cs

```diff
@@ -4,42 +4,36 @@ namespace Nure.Configuration
 {
     public class NureOptions
     {
-        private bool DEFAULT_ALLOW_PRELEASE_DEPENDENCIES = false;
         private const string DEFAULT_PULL_REQUEST_TITLE = "[NuRe] Dependency update";
         private const string DEFAULT_COMMIT_MESSAGE_PREFIX = "Update {} to {}";
-
-        public NureOptions()
-        {
-            AllowPrereleaseDependencies = DEFAULT_ALLOW_PRELEASE_DEPENDENCIES;
-            CustomNugetSources = new List<string>();
-            PackagesToIgnore = new List<string>();
-            CommitMessage = DEFAULT_COMMIT_MESSAGE_PREFIX;
-            PullRequestTitle = DEFAULT_PULL_REQUEST_TITLE;
-        }
+        private const string DEFAULT_COMMIT_USER = "Jenkins";
+        private const string DEFAULT_COMMIT_EMAIL = "email@email.com";
```

Comment:
Does this matter at all or is it a placeholder?

Reply:
Placeholder to support existing lure configs 👍